pkg/sdk/sdk_grpc.pb.go

```diff
@@ -16,7 +16,7 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
 // versions:
 // - protoc-gen-go-grpc v1.2.0
-// - protoc             v3.21.6
+// - protoc             v3.21.12
```

Comment:
🤔  wondering why this is here? (and in the other sdk files)

Reply:
That's a good question, and one I can't answer. I don't have a standalone install of protoc on my machine. Something I did notice is that some of the protobuf seems to have not been generated for some client libraries(if you run it on main, it fails) so this may be an artifact of that?

Reply:
Ooh I see - this is an artifact of running the `make gen-allocation-grpc`? (I would have thought it would have only happened for `make gen-sdk-grpc` 🤔  weird.

I mean it's a change in a comment - not the end of the world 🤷🏻 weird though.